### DISCUSSION TITLE
Pancake on Aptos

### DISCUSSION THREAD

**Discussion by degenSumi** [aptos-labs/aptos-developer-discussions#110] [Questions] [answered]
Hey guys,
I am trying to see this address on the devnet explorer to interact with it(its pancake )
0f9d24010ad96659ee980598ff3848911253bda014e8fe59ce40e9eed9f6585a
but i can't see this in the explorer,
is this the correct module address of pancake swap on aptos devnet?

**Comment by kshitijc1506** [comment]
This account doesn't exist on Devnet. Devnet also does a reset periodically. 

Could you tell me where you got this information from (pancake module address on Devnet)? 

Also, you can view their modules on Mainnet on the explorer [here](https://explorer.aptoslabs.com/account/0xc7efb4076dbe143cbcd98cfaaa929ecfc8f299203dfff63b95ccb6bfe19850fa/modules/code/math?network=mainnet).

**Comment by degenSumi** [comment]
Got it on testnet, needed for testing

**Comment by degenSumi** [comment]
const payload = {
    function: "0xc7efb4076dbe143cbcd98cfaaa929ecfc8f299203dfff63b95ccb6bfe19850fa::router::get_amount_in",
    typeArguments: [parseTypeTag(APTOS_COIN), parseTypeTag(DEVNET_BNB)],
    functionArguments: [new U64(APTOS_COIN_AMOUNT)],

  };

  const amount_in = (await aptos.view({ payload }));
  console.log(amount_in);
Meanwhile @kshitjee  how do i call this method?
Anyone from aptos?

**Reply by kshitijc1506** [reply, marked as answer]
answered [here](https://github.com/aptos-labs/aptos-developer-discussions/discussions/111) for curious devs
